### ISSUE TITLE
Chrome on Window display scroll bar in the bottom list

### ISSUE THREAD

**Issue by noveni** [noveni/product-viewer#1]
![image](https://user-images.githubusercontent.com/10343519/43831584-e7b7b936-9b04-11e8-8069-d99d2b4962be.png)

**Comment by denisflorkin** [comment]
yeah most likely just a overflow-hidden rule missing to not have the vertical scroll bar. altho if we keep this horizontally scrolling container, I think the horizontal scroll bar will not be bad for windows user, they're used to it I think and it actually gives information so good form on the horizontal one

**Comment by noveni** [comment]
👍 
Ok for the horizontal on windows, there are use to. 
I'm gonna fix the vertical.
